cirq-core/cirq/ops/gateset_test.py

```diff
@@ -416,3 +416,12 @@ def test_gateset_eq():
             )
         )
     )
+
+
+def test_gateset_contains_with_tags_to_ignore():
+    gs = cirq.Gateset(
+        cirq.GateFamily(cirq.ZPowGate, tags_to_accept=["PhysicalZTag"]),
```

Comment:
Can we add a separate test for the `tags_to_accept` behavior only?

The following is also failing with assertion error in master:
```python
import cirq
import cirq_google as cg

gs = cirq.Gateset(
    cirq.GateFamily(cirq.ZPowGate, tags_to_accept=[cg.PhysicalZTag()]),
)
op = cirq.Z(cirq.LineQubit(1))
print("op in gs =", op in gs)
```

Reply:
A gateset with only tags_to_ignore is similarly failing on master.